### ISSUE TITLE
Even bigger laser - rebuild and commissioning

### ISSUE THREAD

**Issue by Sean-anotherone** [DoESLiverpool/somebody-should#955]
The "even bigger laser" remains a doorstop and would be a useful addition to the DoES portfolio.

Discrete tasks in order would be :

a) location and orientation to be determined / agreed
b) bulk relocation & reassembly
c) re-commission the machine, including replacing control electronics as necessary (tbc)

**Comment by DoESsean** [comment]
Would the massive laser fit where the woodworking bench is? I've not
measured, so could be very wrong, but it looks like it would be too big,
especially for all three lasers to be in simultaneous use comfortably.

**Comment by johnmckerrell** [comment]
We definitely want to get it going and appreciate any help in doing this! Are the measurements of it somewhere online? Someone handy with Inkscape could also have a go. 

--

**Comment by magman2112** [comment]
As I expect I will be involved quite a bit in getting this laser bag together again, then hopefully working, I have assigned myself to this issue as well.

**Comment by Sean-anotherone** [comment]
:)

**Comment by MatthewCroughan** [comment]
As requested by @Sean-anotherone here is the layout, as drawn by him. I've also created this over on nextcloud with draw.io so we can all collaborate and submit changes.

https://nextcloud.matthewcroughan.co.uk/index.php/s/Pm7oFEMNL2FRZDX

![image](https://user-images.githubusercontent.com/26458780/47975259-ff764700-e0a4-11e8-9c48-4ef11556285c.png)

![img_20181101_224028](https://user-images.githubusercontent.com/26458780/47975233-e5d4ff80-e0a4-11e8-80bf-7c1da1dd00f6.jpg)

**Comment by Robotorium** [comment]
How wide will the aisles be?

Op ma 5 nov. 2018 02:46 schreef MatthewCroughan <notifications@github.com:

**Comment by MatthewCroughan** [comment]
No decision yet, morris isn't going to be sitting in the center of the room, that's just a placeholder to show how large it is until we all decide.

**Comment by johnmckerrell** [comment]
The entrance on that diagram looks a bit big? The photo looks more reasonable but would mean people couldn’t leave piles of wood in there any more (yay!)

--

**Comment by Sean-anotherone** [comment]
Made a good start on this over the weekend with @magman2112.  The woodwork bench and associated dust-raising machinery is all relocated into the CNC room, revealing a that is big enough to get the big laser re-assembled (first) and then explored for functionality and a shopping/wish list raised to get it commissioned.  

We also discovered that the machine does not use a large semiconductor laser as thought, it is a 50W CO2 laser, but an air cooled (possibly RF excited) version - this would likely be expensive to replace if it's dead, but they can be regassed and UK suppliers seem to offer this service.

The next step is to tidy up the stack of wood and sheet material in the laser room and then physically relocate the machine - help will be required as it is HEAVY !

It will likely be necessary to move Gerald to the other end of the room for a few minutes while feeding the bigger machine through the doorway, plus, @magman2112 and I are proposing rotating Gerald by 90° left, making the entrance to the room wider (and no more or less congested than it already is).

**Comment by magman2112** [comment]
It appears that this issue has not been raised with all of the members of the laser maintenance team, so HI Folks
@DoESLiverpool/laser-maintenance 

We are planning a Laser Maintenance day this coming Saturday, where we will attempt to play Musical Laser cutters and get Morris moved into it's hopefully permanent home. This will need a fair few people as this beast is heavy. We will also need to move some stuff out of the room for a while to give us some working room.

It may also be possible to line all three of the laser cutters up on the window side of the room, this is to be confirmed on the day.

I will raise a call on the DoES Google Group as well, so that we get enough people in to make this move hopefully.

**Comment by johnmckerrell** [comment]
I spotted this was in the calendar for 2nd March, guessing that’s not right?

**Comment by magman2112** [comment]
Hi John,

 

I asked Jackie to book this out on Sunday for me (as I don’t have access to this calendar as yet). I guess she accidently picked the wrong month.

 

Could you possibly move this to this Saturday please?

 

Regards

 

Mike

**Comment by johnmckerrell** [comment]
Done

**Comment by KarlDunkerley** [comment]
I can definitely do Saturday but not sunday. What time do we need to assemble?
Cheers, 

Karl Dunkerley

Mob 07710 353 122

eml karldunkerley@yahoo.com 

    On Tuesday, 29 January 2019, 16:12:35 GMT, John McKerrell <notifications@github.com> wrote:  
 
 Done

**Comment by magman2112** [comment]
I was thinking from about 10 to 10:30, though I am open to suggestions if people want to meet earlier or later.

There will be plenty to do with clearing and cleaning tasks anyway, before we need the key group of people to help with the heavy lifting though, so we can be reasonably flexible with timings methinks.

**Comment by KarlDunkerley** [comment]
In my diary. See you then.
Cheers, 

Karl Dunkerley

Mob 07710 353 122

eml karldunkerley@yahoo.com 

    On Tuesday, 29 January 2019, 18:19:08 GMT, Mike Gorman <notifications@github.com> wrote:  
 
 
I was thinking from about 10 to 10:30, though I am open to suggestions if people want to meet earlier or later.

There will be plenty to do with clearing and cleaning tasks anyway, before we need the key group of people to help with the heavy lifting though, so we can be reasonably flexible with timings methinks.

**Comment by Sean-anotherone** [comment]
I'll be having both kids this weekend so I'm out.
Sean

    On Tuesday, 29 January 2019, 16:04:46 GMT, Mike Gorman <notifications@github.com> wrote:  
 
 
It appears that this issue has not been raised with all of the members of the laser maintenance team, so HI Folks
@DoESLiverpool/laser-maintenance

We are planning a Laser Maintenance day this coming Saturday, where we will attempt to play Musical Laser cutters and get Morris moved into it's hopefully permanent home. This will need a fair few people as this beast is heavy. We will also need to move some stuff out of the room for a while to give us some working room.

It may also be possible to line all three of the laser cutters up on the window side of the room, this is to be confirmed on the day.

I will raise a call on the DoES Google Group as well, so that we get enough people in to make this move hopefully.

**Comment by zarino** [comment]
@magman2112 It looks like I’m free this Saturday, so count me in 👍

**Comment by magman2112** [comment]
Morris was successfully moved into the laser room on Saturday and largely re-assembled. There are still a few parts of the structure to be refitted, then we can start looking at some basic tests, such as whether the laser is working for example.

**Comment by magman2112** [comment]
Most of the parts have now been fitted to Morris and initial test have shown that the laser is working and the machine is able to cut test patterns using the built in firmware.

There appears to be a missing joint from the extraction system at the moment though, so there is still some work to be done to make this aspect work.

Discussions about the software and controller issues are in thread #607.

**Comment by magman2112** [comment]
With the new laser controller here, the electronics side is now progressing.

On the rebuild side, some work is needed on the lid for the laser, as this is quite badly scratched. possibly t-cutting or finding some perspex polish would be a good start.

**Comment by MatthewCroughan** [comment]
Do you think this can get done on the hack the space day 29/12/2019? @magman2112 If all that remains is the software and wiring issues, we can aim for it.

**Comment by magman2112** [comment]
@Sean-anotherone and I have identified some issues with one of the servo motors which we haven’t found a solution fo so far, so this is a bottleneck to getting this machine working. 

I suspect that there will be other priorities on the hack the space day, but this still might be worth looking at.

**Comment by amcewen** [comment]
You are allowed to share the problems you've hit here, y'know ;-)  You never know what knowledge/experience there is to tap into to help...

**Comment by Sean-anotherone** [comment]
The servo drives in one direction without control input, even with the controller disconnected.  Next step is to apply external power to the encoder and scope the output to see if it is providing spurious signals.   The odd thing is that the servo movement is intermittent stop/start but always in the same direction.  It is the y servo, X works fine, we have already eliminated the controller by substitution, issue is with the encoder or cabling.  Y servo is currently part dismantled.

Other issue with the machine is that the 4 A breaker on the 24 v supplies is fragile, needs a 6 type c.

Sent from Yahoo Mail on Android 
 
  On Fri, 27 Dec 2019 at 11:42, Adrian McEwen<notifications@github.com> wrote:   
You are allowed to share the problems you've hit here, y'know ;-) You never know what knowledge/experience there is to tap into to help...

**Comment by magman2112** [comment]
Latest update. 

The problematic servo drive has now been rewired. Initially we had a similar problem as before the rewire, but when we broke the common screen, the servo started to behave itself.

@Sean-anotherone then did a bit of tuning on the servo drives and both axes are now moving in reasonable order.

We then downloaded the Laser Burn trial software and successfully controlled the laser with the software over the USB connection. We had a bit of fun getting it to talk to the PC for Gerald, so we have only tested using my laptop at the moment.

Next we checked the accuracy of movement on the laser access using a dial gauge the Sean brought in. with a bit of adjustment, bit axes are now calibrated for correct movement.

Finally, we worked to get control of the laser from the new controller. After quite a bit of work, Sean managed to get this working and successfully cut a hexagon in paper - the first output of the new machine configuration. Further testing though found that the laser is considerably down on power - not being able to cut 3mm ply wood on full power.

So basically, the laser is working but is down on power.

We still need to fit limit switches to the axes though (for homing as well) and the extract system needs to be linked up - which we plan to start on next Maker Night.

**Comment by magman2112** [comment]
I spoke to Paul Mason at Millenium Laser Ltd on Thursday about the possibility of re-gassing the laser for Monty.

I was informed that it would likely cost £1400 to re-gas the Universal Laser laser tube for Monty. This is obviously too much for us to spend at present, but I appear to have made a reasonable contact should we look to replace the current laser with an alternate - such as a tube similar to Sophia's in the future.

In the short term, I think we should investigate of Monty is OK to use for engraving or paper cutting using it's current power level. Then we can evaluate the demand and whether it is worth  getting the current laser re-gassed.

For the record, here are the specific details for the current laser in Monty.

Universal Laser UL-50-OEM
Serial Number OEM#: RP-10,562C
Date: March 2008.

**Comment by KarlDunkerley** [comment]
Do you know the spec of the tube? I have a spare one for mine. Only 50W but working. Might be showhornable.
Welcome to use it until can get a proper one.
Also seems v high price. There is a chap in North Wales who can do it I believe. Lost his details but he was on Facebook.


    On Thursday, 16 January 2020, 15:45:17 GMT, Mike Gorman <notifications@github.com> wrote:  
 
 
I spoke to Paul Mason at Millenium Laser Ltd on Thursday about the possibility of re-gassing the laser for Monty.

I was informed that it would likely cost £1400 to re-gas the Universal Laser laser tube for Monty. This is obviously too much for us to spend at present, but I appear to have made a reasonable contact should we look to replace the current laser with an alternate - such as a tube similar to Sophia's in the future.

In the short term, I think we should investigate of Monty is OK to use for engraving or paper cutting using it's current power level. Then we can evaluate the demand and whether it is worth getting the current laser re-gassed.

For the record, here are the specific details for the current laser in Monty.

Universal Laser UL-50-OEM
Serial Number OEM#: RP-10,562C
Date: March 2008.

**Comment by KarlDunkerley** [comment]
I think his name was Richard Butler.
    On Thursday, 16 January 2020, 15:45:17 GMT, Mike Gorman <notifications@github.com> wrote:  
 
 
I spoke to Paul Mason at Millenium Laser Ltd on Thursday about the possibility of re-gassing the laser for Monty.

I was informed that it would likely cost £1400 to re-gas the Universal Laser laser tube for Monty. This is obviously too much for us to spend at present, but I appear to have made a reasonable contact should we look to replace the current laser with an alternate - such as a tube similar to Sophia's in the future.

In the short term, I think we should investigate of Monty is OK to use for engraving or paper cutting using it's current power level. Then we can evaluate the demand and whether it is worth getting the current laser re-gassed.

For the record, here are the specific details for the current laser in Monty.

Universal Laser UL-50-OEM
Serial Number OEM#: RP-10,562C
Date: March 2008.

**Comment by KarlDunkerley** [comment]
info@rtbtech.co.uk

    On Thursday, 16 January 2020, 15:45:17 GMT, Mike Gorman <notifications@github.com> wrote:  
 
 
I spoke to Paul Mason at Millenium Laser Ltd on Thursday about the possibility of re-gassing the laser for Monty.

I was informed that it would likely cost £1400 to re-gas the Universal Laser laser tube for Monty. This is obviously too much for us to spend at present, but I appear to have made a reasonable contact should we look to replace the current laser with an alternate - such as a tube similar to Sophia's in the future.

In the short term, I think we should investigate of Monty is OK to use for engraving or paper cutting using it's current power level. Then we can evaluate the demand and whether it is worth getting the current laser re-gassed.

For the record, here are the specific details for the current laser in Monty.

Universal Laser UL-50-OEM
Serial Number OEM#: RP-10,562C
Date: March 2008.

**Comment by Sean-anotherone** [comment]
Good summary from Mike.

The comments from Karl are also valid - I think there is a spare / 
repairable laser PSU about the place, and even a new sophia-sized tube 
is not too expensive, so backfitting a HV tube laser would be very 
doable and would give a very-large-bed 50W machine that would allow 
either Sophia or Gerald to be taken offline for their (well overdue) 
essential maintenance.

Or, possibly a bit contentiously, we could "sacrifice" Sophia 
(temporarily?) and fit that tube/PSU/chiller on Monty - it would still 
be the same power & cost to run, but advantaging the large bed, top 
quality optical path* and new software & controller.  Sophia could then 
get overhauled in slow time to fix all the sloppy axis issues.


Sean-anotherone

(*) The mirrors on Monty are not 100%, but the quality of the mounts is 
WAAAYYYY better than either of the other machines.

**Comment by Sean-anotherone** [comment]
As requested - the prices of a few options

1. Regas the RF laser - £1400

2. Find a buyer for the (working but low on power) RF laser, they are £8000 new, so it should be worth >£1000, especially in parts credit.  Retrofit a tube laser like the list below.

3. Just buy and fit a tube laser:

60W tube ebay  (cannot be more than 1M long) - £92
https://www.ebay.co.uk/itm/Ridgeyard-60W-CO2-Laser-Tube-Length-1000mm-for-60W-Laser-Engraver-Cutter-Machine/113982815223?hash=item1a89e7bbf7:g:RNYAAOSwl2Bd16L5

60WPSU ebay - £75
https://www.ebay.co.uk/itm/Ridgeyard-60W-CO2-Laser-Tube-Length-1000mm-for-60W-Laser-Engraver-Cutter-Machine/113982815223?hash=item1a89e7bbf7:g:RNYAAOSwl2Bd16L5

And a chiller - ebay £100 for a fan box, (£300 for an actual chilly chiller)

Ie- ~£270 plus odds & sods (like mounts, tubing etc.)

might do better with a bit of shopping around.  The biggest tube we could easily fit is about  1M

**Comment by Sean-anotherone** [comment]
Following some words of wisdom from @goatchurchprime, @magman2112 and I got both axes on Monty behaving properly, but the calibration needs to be repeated. (I'll bring the gauge this week)

The latest odd behaviour relates to the machine engraving (weakly but) well, with pwm greyscale, but not giving any laser output in line/cut mode???????

Very odd, and confusing.  Will be prodding it with oscilloscope on Thurs to see if the pwm has the expected w.

**Comment by KarlDunkerley** [comment]
My spare 50w is available for testing purposes if you need it. Will be in on Thursday night. Can bring it in. Let me know asap.
    On Monday, 27 January 2020, 18:19:59 GMT, Sean Jarman <notifications@github.com> wrote:  
 
 
Following some words of wisdom from @goatchurchprime, @magman2112 and I got both axes on Monty behaving properly, but the calibration needs to be repeated. (I'll bring the gauge this week)

The latest odd behaviour relates to the machine engraving (weakly but) well, with pwm greyscale, but not giving any laser output in line/cut mode???????

Very odd, and confusing. Will be prodding it with oscilloscope on Thurs to see if the pwm has the expected w.

**Comment by Sean-anotherone** [comment]
Hi Karl, not needed for now thanks, the machine has an RF laser currently so the supply etc would all need changing.  Options still being explored before going down the retrofit route.

Sent from Yahoo Mail on Android 
 
  On Tue, 28 Jan 2020 at 13:11, KarlDunkerley<notifications@github.com> wrote:    My spare 50w is available for testing purposes if you need it. Will be in on Thursday night. Can bring it in. Let me know asap.
 On Monday, 27 January 2020, 18:19:59 GMT, Sean Jarman <notifications@github.com> wrote: 
 
 
Following some words of wisdom from @goatchurchprime, @magman2112 and I got both axes on Monty behaving properly, but the calibration needs to be repeated. (I'll bring the gauge this week)

The latest odd behaviour relates to the machine engraving (weakly but) well, with pwm greyscale, but not giving any laser output in line/cut mode???????

Very odd, and confusing. Will be prodding it with oscilloscope on Thurs to see if the pwm has the expected w.

**Comment by magman2112** [comment]
Latest tests on Thursday showed that the current laser is worse than we thought.

First, on a positive note, @Sean-anotherone adjusted the servo drive for the X-axis and we are now getting square cuts and engraving, not trapezoid as we were getting before. 

The main outstanding problem though is with the laser output power. Running line cut tests showed that the laser will only cut for a few seconds at 100%, before the power fades away. It will also cut at lower power settings, but again power tails off the longer the laser is cutting. This would seem to suggest a more significant fault with the laser - but could still be down to just needing re-gassing. This does remind me of similar issues with Gerald and Sophia when the laser tube got close to end of life.

So it looks like a test with a Sophia type laser tube is the likely way forward with Monty at present. So @KarlDunkerley, we will take you up on your offer to borrow your spare laser tube. We do have a couple of spare laser power supplies at present, but one is definitely in need of repair and the other looks used so needs to be tested. We will also have to improvise some cooling for the laser tube and make new mounting brackets for the tube.

So currently one step forward and a couple backwards unfortunately.

**Comment by amcewen** [comment]
Bummer about the laser, but thanks for the continued work on it and also for the updates here @Sean-anotherone and @magman2112

**Comment by KarlDunkerley** [comment]
No problem. How soon do you need it? Should be in later this week to work on the heat press but waiting for an oversized dye sub print to arrive. Alternatively, if you need it sooner I can meet up with Mike G somewhere or drop it off at your house.
Let me know.


    On Monday, 3 February 2020, 11:47:08 GMT, Mike Gorman <notifications@github.com> wrote:  
 
 
Latest tests on Thursday showed that the current laser is worse than we thought.

First, on a positive note, @Sean-anotherone adjusted the servo drive for the X-axis and we are now getting square cuts and engraving, not trapezoid as we were getting before.

The main outstanding problem though is with the laser output power. Running line cut tests showed that the laser will only cut for a few seconds at 100%, before the power fades away. It will also cut at lower power settings, but again power tails off the longer the laser is cutting. This would seem to suggest a more significant fault with the laser - but could still be down to just needing re-gassing. This does remind me of similar issues with Gerald and Sophia when the laser tube got close to end of life.

So it looks like a test with a Sophia type laser tube is the likely way forward with Monty at present. So @KarlDunkerley, we will take you up on your offer to borrow your spare laser tube. We do have a couple of spare laser power supplies at present, but one is definitely in need of repair and the other looks used so needs to be tested. We will also have to improvise some cooling for the laser tube and make new mounting brackets for the tube.

So currently one step forward and a couple backwards unfortunately.

**Comment by Sean-anotherone** [comment]
Jackie asked for an update on the latest position in time for the meeting this weekend, so here it is:

1.  The "machine" is working very well in terms of motors, axes, mirrors etc.  The servos have been calibrated as well as we can without being able to cut test shapes and seem to be good to within about 0.02mm.
2. The laser continues to behave exactly as described above by Mike , a couple of minutes of low power, followed by fade-out to zero power.

Options :

a) backfit a CO2 tube laser - medium costs, limitations on lifetime but would then be the best machine available (within the power of the tube).  It _may_ be possible to re-engineer the machine to take a Gerald-sized tube, but that would be much more work.

b) get the RF laser refurbed - £1400 if it works (it might not be repairable) but it would last a long long time.

c) strip the whole machine for parts and upgrade Gerald ( this would take several days of effort)

d) Attempt a fix on the RF laser - there is a possibility that it has been knocked out of alignment.  This would involve careful adjustments to the back mirror of the resonator, it might work, or it might end in a laser that can't even be sold-on for store credit - I would be prepared to have a go at this but only with a very clear mandate that the risk is accepted.  Given that the laser was lying about for 2 years and that it drifts off as it warms up, a bit of misalignment that worsens as the metal expands is not an unreasonable diagnosis.  Or I might destroy the optical resonator completely, hard to tell without trying.

e) try to trade-in the RF laser for the parts needed to pursue option (a), someone would need to take a day-trip to South Wales. (fuel probably cheaper than the postage would be !)


Apart from the laser, and some further reassembly, all that is needed to finish month is a few O-rings for the mirror adjusters a PC to drive it from ad some operator training (the software is very intuitive so conversion would not take long).  Future work would include making and fitting limit and home switches, but we live without these on Gerald, so not urgent.

Recommendations - in order of my personal preference :

(e)
(a)
(d)
(b)
(c)

**Comment by jackie1050** [comment]
Thanks @Sean-anotherone<https://github.com/Sean-anotherone> !

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by KarlDunkerley** [comment]
I'm going to pass my spare CO2 tube on to Mike G to see if that works. Hopefully that will inform the way forward.
Just don't blow it up LOL
    On Saturday, 8 February 2020, 10:31:14 GMT, Sean Jarman <notifications@github.com> wrote:  
 
 
Jackie asked for an update on the latest position in time for the meeting this weekend, so here it is:
   
   - The "machine" is working very well in terms of motors, axes, mirrors etc. The servos have been calibrated as well as we can without being able to cut test shapes and seem to be good to within about 0.02mm.
   - The laser continues to behave exactly as described above by Mike , a couple of minutes of low power, followed by fade-out to zero power.

Options :

a) backfit a CO2 tube laser - medium costs, limitations on lifetime but would then be the best machine available (within the power of the tube). It may be possible to re-engineer the machine to take a Gerald-sized tube, but that would be much more work.

b) get the RF laser refurbed - £1400 if it works (it might not be repairable) but it would last a long long time.

c) strip the whole machine for parts and upgrade Gerald ( this would take several days of effort)

d) Attempt a fix on the RF laser - there is a possibility that it has been knocked out of alignment. This would involve careful adjustments to the back mirror of the resonator, it might work, or it might end in a laser that can't even be sold-on for store credit - I would be prepared to have a go at this but only with a very clear mandate that the risk is accepted. Given that the laser was lying about for 2 years and that it drifts off as it warms up, a bit of misalignment that worsens as the metal expands is not an unreasonable diagnosis. Or I might destroy the optical resonator completely, hard to tell without trying.

e) try to trade-in the RF laser for the parts needed to pursue option (a), someone would need to take a day-trip to South Wales. (fuel probably cheaper than the postage would be !)

Apart from the laser, and some further reassembly, all that is needed to finish month is a few O-rings for the mirror adjusters a PC to drive it from ad some operator training (the software is very intuitive so conversion would not take long). Future work would include making and fitting limit and home switches, but we live without these on Gerald, so not urgent.

Recommendations - in order of my personal preference :

(e)
(a)
(d)
(b)
(c)

**Comment by johnmckerrell** [comment]
FYI I think the guy we're talking about in Wales is actually North Wales, Prestatyn. At least if we're talking about the same person:

Richard Butler

RTBTech ltd
www.rtbtech.co.uk

Last time we talked to him he said:

> Our callout fee to Liverpool is £380 including servicing one machine, we then charge an additional £180 for an other machines.

**Comment by amcewen** [comment]
I've pinged Keith from Leather Satchel company to see who he uses for servicing (as he's got one of those laser-cutters)

**Comment by johnmckerrell** [comment]
Discussed in the directors meeting and we're tending towards getting the RF laser re-gassed, assuming it’s around the cost we’ve been discussing so far.

**Comment by jackie1050** [comment]
I've emailed Jan at Hope Uni to ask who they get to service their similar laser cutters

**Comment by KarlDunkerley** [comment]
Is it worth brining in my CO2 tube in that case? Seem to think it would mean some re-engineering so is it better to wait until decision made?
Karl
    On Sunday, 9 February 2020, 12:47:24 GMT, John McKerrell <notifications@github.com> wrote:  
 
 
Discussed in the directors meeting and we're tending towards getting the RF laser re-gassed, assuming it’s around the cost we’ve been discussing so far.

**Comment by amcewen** [comment]
Keith does most of the maintenance in house, but for a regas would talk to GS Lasers (the manufacturer).  So we could ask them for a regas quote (and see if RTBTech deal with this sort of laser too, as it doesn't look like we've got a regas quote from them either?)

**Comment by BornLuckiest** [comment]
Also, just let me know if there's anything I can do to help :)

**Comment by Sean-anotherone** [comment]
Someone who can make the phone calls during the 9-5 window of opportunity needs to grasp the nettle on this and get a few more quotes

**Comment by jackie1050** [comment]
I seem to have missed adding Jan Egerton's response "We use HPC lasers and they are based in saddleworth and will service and are reasonable. They have a good customer service over the phone". We've dealt with them before - might be worth getting a quote.

**Comment by johnmckerrell** [comment]
@DoESsean can you take a look at this and see if you can work out the next step, get prices for regassing the laser tube.

**Comment by Sean-anotherone** [comment]
*bump*

**Comment by magman2112** [comment]
Time for an update after quite a bit more research.

First, I did some research on the existing Universal Laser UL-50 unit. 

Laserrefill.com stated on their website that the UL Series lasers should be good for 20,000-30,000 hours of use. But the key thing is that the CO2 gas in the tube can leak past the seals, so whether you are using the laser of not, the gas eventually leaks out. This is also temperature dependant, so in cold weather, it can leak more.

I looked for alternate refill services, but these were mostly in the US. Normal websites tend not to list refill prices, but I did find some on eBay. One user (olafscaley) was chargingh $950 for a refill, but with postage and Import charges this eventually worked out at £1080.86. Another user on eBay had an apparently cheaper refill. at $950 (laser-tech), but with postage this worked out at £1068.18, plus any additional import duties.

I did find one user on eBay selling new UL50 lasers, at $2900 each, but with carriage this worked out to £2307.78 - and again, there would be import duties to add to this.

Next I spoke to Cad-Cam Technology, the original manufacturer of the whole laser machine. They told me that Universal Laser are now no longer reprocessing the UL-50 laser, but a refill from them would have been in the region of £2500. Also, Cad-Cam Technology is now under new ownership, with the laser sales now under Summer, a Belgian company. They did give an alternate contact though - for Anglian Management Services, this company manages the education sales, which is where I believe our original laser came from.

I had a good conversation with David Barkham of Anglia Management Services (07872 944605). He stated that his company is the educational supplier for these laser cutters. He also stated that Universal Laser discontinued support for the UL-50 lasers from February this year so a replacement would be needed from now on. The replacement is the ULR range, from Universal Laser, but an equivalent ULR50 laser would be in the region of £5500.

The last conversation I had was with Paul Mason of Millenium Laser. Paul again stated that there are some concerns over the long term reliability of the UL Series lasers from Universal Laser. He can still do a regass of these lasers, but this would cost £1450 and he would only give us a 6 months warranty on this refill.

We also discussed a possible replacement, using a glass high voltage DC laser, like those used in Gerald and Sophia. Millenium Laser also sell these - a 60W tube with Power Supply would cost £945, though they do give a 2 year warranty on these tubes that the power won't drop more than 15% - these tubes are also capable of 70-75W operation from new. This tube is 1m long, so should fit Monty, and would also need a chiller and associated plumbing as well as us building any required bracketry to install the tube. 

A third option would be a replacement RF laser, which Paulk sells, but I didn't ask about this option as Paul already mentioned that this would cost more than a refill of the old tube.

I also asked if there was any value in the old UL-50 laser. Paul said it was of no value to him, as well as other people in the industry, though he suggested that we could possibly list it on eBay were we might get a £100 or more for the unit. We would have to list is as needing a refill or spares or repair.


So given all of this new data - it looks like the old Universal Laser UL-50 RF laser is going to be a non-starter and we should investigate replacing this with a High Voltage DC glass tube laser and ancilliaries.

**Comment by KarlDunkerley** [comment]
Would the existing chiller be able to handle both machines and save some pennies?
    On Thursday, 6 August 2020, 18:21:38 BST, Mike Gorman <notifications@github.com> wrote:  
 
 


Time for an update after quite a bit more research.

First, I did some research on the existing Universal Laser UL-50 unit.

Laserrefill.com stated on their website that the UL Series lasers should be good for 20,000-30,000 hours of use. But the key thing is that the CO2 gas in the tube can leak past the seals, so whether you are using the laser of not, the gas eventually leaks out. This is also temperature dependant, so in cold weather, it can leak more.

I looked for alternate refill services, but these were mostly in the US. Normal websites tend not to list refill prices, but I did find some on eBay. One user (olafscaley) was chargingh $950 for a refill, but with postage and Import charges this eventually worked out at £1080.86. Another user on eBay had an apparently cheaper refill. at $950 (laser-tech), but with postage this worked out at £1068.18, plus any additional import duties.

I did find one user on eBay selling new UL50 lasers, at $2900 each, but with carriage this worked out to £2307.78 - and again, there would be import duties to add to this.

Next I spoke to Cad-Cam Technology, the original manufacturer of the whole laser machine. They told me that Universal Laser are now no longer reprocessing the UL-50 laser, but a refill from them would have been in the region of £2500. Also, Cad-Cam Technology is now under new ownership, with the laser sales now under Summer, a Belgian company. They did give an alternate contact though - for Anglian Management Services, this company manages the education sales, which is where I believe our original laser came from.

I had a good conversation with David Barkham of Anglia Management Services (07872 944605). He stated that his company is the educational supplier for these laser cutters. He also stated that Universal Laser discontinued support for the UL-50 lasers from February this year so a replacement would be needed from now on. The replacement is the ULR range, from Universal Laser, but an equivalent ULR50 laser would be in the region of £5500.

The last conversation I had was with Paul Mason of Millenium Laser. Paul again stated that there are some concerns over the long term reliability of the UL Series lasers from Universal Laser. He can still do a regass of these lasers, but this would cost £1450 and he would only give us a 6 months warranty on this refill.

We also discussed a possible replacement, using a glass high voltage DC laser, like those used in Gerald and Sophia. Millenium Laser also sell these - a 60W tube with Power Supply would cost £945, though they do give a 2 year warranty on these tubes that the power won't drop more than 15% - these tubes are also capable of 70-75W operation from new. This tube is 1m long, so should fit Monty, and would also need a chiller and associated plumbing as well as us building any required bracketry to install the tube.

A third option would be a replacement RF laser, which Paulk sells, but I didn't ask about this option as Paul already mentioned that this would cost more than a refill of the old tube.

I also asked if there was any value in the old UL-50 laser. Paul said it was of no value to him, as well as other people in the industry, though he suggested that we could possibly list it on eBay were we might get a £100 or more for the unit. We would have to list is as needing a refill or spares or repair.

So given all of this new data - it looks like the old Universal Laser UL-50 RF laser is going to be a non-starter and we should investigate replacing this with a High Voltage DC glass tube laser and ancilliaries.

**Comment by Sean-anotherone** [comment]
We thought about this earlier - probably, but as Mike pointed out it would be a single point of failure that would take out both machines.
Sea
    On Thursday, 6 August 2020, 19:24:38 BST, KarlDunkerley <notifications@github.com> wrote:  
 
 

 Would the existing chiller be able to handle both machines and save some pennies?
 On Thursday, 6 August 2020, 18:21:38 BST, Mike Gorman <notifications@github.com> wrote: 
 
 


Time for an update after quite a bit more research.

First, I did some research on the existing Universal Laser UL-50 unit.

Laserrefill.com stated on their website that the UL Series lasers should be good for 20,000-30,000 hours of use. But the key thing is that the CO2 gas in the tube can leak past the seals, so whether you are using the laser of not, the gas eventually leaks out. This is also temperature dependant, so in cold weather, it can leak more.

I looked for alternate refill services, but these were mostly in the US. Normal websites tend not to list refill prices, but I did find some on eBay. One user (olafscaley) was chargingh $950 for a refill, but with postage and Import charges this eventually worked out at £1080.86. Another user on eBay had an apparently cheaper refill. at $950 (laser-tech), but with postage this worked out at £1068.18, plus any additional import duties.

I did find one user on eBay selling new UL50 lasers, at $2900 each, but with carriage this worked out to £2307.78 - and again, there would be import duties to add to this.

Next I spoke to Cad-Cam Technology, the original manufacturer of the whole laser machine. They told me that Universal Laser are now no longer reprocessing the UL-50 laser, but a refill from them would have been in the region of £2500. Also, Cad-Cam Technology is now under new ownership, with the laser sales now under Summer, a Belgian company. They did give an alternate contact though - for Anglian Management Services, this company manages the education sales, which is where I believe our original laser came from.

I had a good conversation with David Barkham of Anglia Management Services (07872 944605). He stated that his company is the educational supplier for these laser cutters. He also stated that Universal Laser discontinued support for the UL-50 lasers from February this year so a replacement would be needed from now on. The replacement is the ULR range, from Universal Laser, but an equivalent ULR50 laser would be in the region of £5500.

The last conversation I had was with Paul Mason of Millenium Laser. Paul again stated that there are some concerns over the long term reliability of the UL Series lasers from Universal Laser. He can still do a regass of these lasers, but this would cost £1450 and he would only give us a 6 months warranty on this refill.

We also discussed a possible replacement, using a glass high voltage DC laser, like those used in Gerald and Sophia. Millenium Laser also sell these - a 60W tube with Power Supply would cost £945, though they do give a 2 year warranty on these tubes that the power won't drop more than 15% - these tubes are also capable of 70-75W operation from new. This tube is 1m long, so should fit Monty, and would also need a chiller and associated plumbing as well as us building any required bracketry to install the tube.

A third option would be a replacement RF laser, which Paulk sells, but I didn't ask about this option as Paul already mentioned that this would cost more than a refill of the old tube.

I also asked if there was any value in the old UL-50 laser. Paul said it was of no value to him, as well as other people in the industry, though he suggested that we could possibly list it on eBay were we might get a £100 or more for the unit. We would have to list is as needing a refill or spares or repair.

So given all of this new data - it looks like the old Universal Laser UL-50 RF laser is going to be a non-starter and we should investigate replacing this with a High Voltage DC glass tube laser and ancilliaries.

**Comment by magman2112** [comment]
On the Maker night call last Thursday, we agreed that it is not economic to proceed with the re-gassing of the the old RF laser tube from Monty.

The next step is to investigate and cost up the parts needed to use a DC High Voltage tube, as used in Gerald and Sophia. We agreed to aim for an 80W tube, so that we can better cut 6mm material.

Here is an initial shopping list:-

- Laser tube
- Laser tube mounts
- Laser HT Power Supply
- Chiller
- Flow Sensor
- HT cabling from PSU to Laser Tube
- Silicone tubing from chiller to tube
- Through panel couplers to route water tubes out of laser cutter
- Miscellaneous cabling

One preference I expressed was to use a similar tube to that fitted to Gerald - which was obtained from HPC Laser in Halifax. this tube is listed on their web shop at £450 + VAT I expect. This is an 80W (95W peak) tube. The main issue with this will be that it is 1600mm long, so we would have to do a lot of work to fit this in Monty, so may not be the best option after all. I could find no details of the guarantee period on the HPC web site either.

I also had a look at Smoke and Mirrors, but the largest tube they appear to sell is the Yongli 1200-G, a 65W (80W peak) tube, which I believe is the type of tube currently fitted to Sophia. This tube is listed for £300 and appears to include VAT and comes with a 3 month guarantee.

I did find an interesting shop from a Google search though - https://www.cloudraylaser.com/

This site lists a large number of differrent tubes as well as power supplies, chillers, tube mounts, etc. so could possiblky be used as almost a one stop shop. As it is abroad though, we will have to pay not inconsiderable shipping and import duty. For example, I found an 75W (90W max) tube that is only 1100mm long for $272. This same tube is listed on their ebay shop for £381.99 including free shipping.

Something to discuss on our next Maker Night call perhaps - especially the tube to go for.

**Comment by KarlDunkerley** [comment]
If it helps I still have my spare 50 watter if that gets it up and running cheaply and can then use it to earn funds to pay for a bigger tube given that money may be tight right now.

Karl
    On Thursday, 13 August 2020, 17:40:28 BST, Mike Gorman <notifications@github.com> wrote:  
 
 


On the Maker night call last Thursday, we agreed that it is not economic to proceed with the re-gassing of the the old RF laser tube from Monty.

The next step is to investigate and cost up the parts needed to use a DC High Voltage tube, as used in Gerald and Sophia. We agreed to aim for an 80W tube, so that we can better cut 6mm material.

Here is an initial shopping list:-
   
   - Laser tube
   - Laser tube mounts
   - Laser HT Power Supply
   - Chiller
   - Flow Sensor
   - HT cabling from PSU to Laser Tube
   - Silicone tubing from chiller to tube
   - Through panel couplers to route water tubes out of laser cutter
   - Miscellaneous cabling

One preference I expressed was to use a similar tube to that fitted to Gerald - which was obtained from HPC Laser in Halifax. this tube is listed on their web shop at £450 + VAT I expect. This is an 80W (95W peak) tube. The main issue with this will be that it is 1600mm long, so we would have to do a lot of work to fit this in Monty, so may not be the best option after all. I could find no details of the guarantee period on the HPC web site either.

I also had a look at Smoke and Mirrors, but the largest tube they appear to sell is the Yongli 1200-G, a 65W (80W peak) tube, which I believe is the type of tube currently fitted to Sophia. This tube is listed for £300 and appears to include VAT and comes with a 3 month guarantee.

I did find an interesting shop from a Google search though - https://www.cloudraylaser.com/

This site lists a large number of differrent tubes as well as power supplies, chillers, tube mounts, etc. so could possiblky be used as almost a one stop shop. As it is abroad though, we will have to pay not inconsiderable shipping and import duty. For example, I found an 75W (90W max) tube that is only 1100mm long for $272. This same tube is listed on their ebay shop for £381.99 including free shipping.

Something to discuss on our next Maker Night call perhaps - especially the tube to go for.

**Comment by Sean-anotherone** [comment]
A good analysis from Mike, however a bit of caution from me...  I think the "line" of the tube falls in the part of the cover that is angled - this leads me to recommend that some careful measuring is necessary before committing.  Alternatively, we could weld up a "pocket" for the back end of the tube to fit into by modding the LH top cover.  Important to get this step right as the end cap is at several kilovolts and we don;t want to either zap operators or trip the circuit if it arcs.
A bigger modification would be needed to fit a Gerald sized tube, but that would give some redundancy?  As there would be an additional mirror in the path this would also allow for a beam combiner to be included along the way (or we could rob the existing one from the front of the RF laser?)

We could always take the offer of the 50W tube from Karl, attempt a repair on the supply that was arcing (it might just need a relacquer?) and use that to provide better confidence in the machine before committing the big bucks?  ( I think the machine is great, but we can't prove that)

We do know for certain that all the rest is fully functional - albeit that the servos need a bit more tuning, so at next opportunity I suggest deinstalling the RF laser and associated gubbins, to enable better measurement of the existing space.  Rather like Gerald, the machine only has a lid interlock and an e-stop.  It could do with homing sensors.

Sean
    On Thursday, 13 August 2020, 17:53:57 BST, KarlDunkerley <notifications@github.com> wrote:  
 
 

 If it helps I still have my spare 50 watter if that gets it up and running cheaply and can then use it to earn funds to pay for a bigger tube given that money may be tight right now.

Karl
 On Thursday, 13 August 2020, 17:40:28 BST, Mike Gorman <notifications@github.com> wrote: 
 
 


On the Maker night call last Thursday, we agreed that it is not economic to proceed with the re-gassing of the the old RF laser tube from Monty.

The next step is to investigate and cost up the parts needed to use a DC High Voltage tube, as used in Gerald and Sophia. We agreed to aim for an 80W tube, so that we can better cut 6mm material.

Here is an initial shopping list:-
 
 - Laser tube
 - Laser tube mounts
 - Laser HT Power Supply
 - Chiller
 - Flow Sensor
 - HT cabling from PSU to Laser Tube
 - Silicone tubing from chiller to tube
 - Through panel couplers to route water tubes out of laser cutter
 - Miscellaneous cabling

One preference I expressed was to use a similar tube to that fitted to Gerald - which was obtained from HPC Laser in Halifax. this tube is listed on their web shop at £450 + VAT I expect. This is an 80W (95W peak) tube. The main issue with this will be that it is 1600mm long, so we would have to do a lot of work to fit this in Monty, so may not be the best option after all. I could find no details of the guarantee period on the HPC web site either.

I also had a look at Smoke and Mirrors, but the largest tube they appear to sell is the Yongli 1200-G, a 65W (80W peak) tube, which I believe is the type of tube currently fitted to Sophia. This tube is listed for £300 and appears to include VAT and comes with a 3 month guarantee.

I did find an interesting shop from a Google search though - https://www.cloudraylaser.com/

This site lists a large number of differrent tubes as well as power supplies, chillers, tube mounts, etc. so could possiblky be used as almost a one stop shop. As it is abroad though, we will have to pay not inconsiderable shipping and import duty. For example, I found an 75W (90W max) tube that is only 1100mm long for $272. This same tube is listed on their ebay shop for £381.99 including free shipping.

Something to discuss on our next Maker Night call perhaps - especially the tube to go for.

**Comment by magman2112** [comment]
I've been working on some costings for a reasonably complete package for Monty.

I priced up the RECI W1 tube for this estimate. Primarily as it has a 10,000 hour life and 18 month guarantee listed, but it is also very short at 1050mm for a 75-90W tube. This is a more expensive tube than some similar 70-80W tubes, but this tube is also very short for this power - then there is the added bonus of the long life.

One issue which I haven't found an answer for so far is how much import duty we are likely to pay when we get these parts, but I would guess at 10-12%, plus a handling charge.

I have looked at prices from Cloudray as a single supplier - but their shipping costs are high, at $360, - so I have also looked at buying the same or similar parts from eBay as well.

I have attached a spreadsheet with my initial workings, but the bottom line comes out at £1100 direct from Cloudbay or approx £880 from multiple sources (there are some small parts missing from this total, about $60 worth on the Cloudray shopping list). So I guess the multiple sources option wins out here - especially as one of the item (the chiller) was sourced in Europe, so no VAT or Import duty. These prices include VAT and estimated import duty and handling charges.

[Monty Rebuild Cost 18AUG2020.xlsx](https://github.com/DoESLiverpool/somebody-should/files/5091845/Monty.Rebuild.Cost.18AUG2020.xlsx)

Interested in thoughts and comments on this estimate.

**Comment by magman2112** [comment]
Oops, slight booboo. 

It appears the eBay RECI tube is the T1, not the W1. This is still a high quality tube, but not quite as good as the W1 - 8000hrs life, 366 days warranty and 1100mm long. Other suppliers of the W1 tube on eBay want over £500 though.

**Comment by magman2112** [comment]
With the decision being taken in the recent Directors meeting to proceed with replacing the tube on Gerald, I have built a shopping list for the parts whic is attached below.

[Monty Rebuild Cost 25AUG2020.xlsx](https://github.com/DoESLiverpool/somebody-should/files/5123420/Monty.Rebuild.Cost.25AUG2020.xlsx)

I will work with @DoESsean on Thursday to get these orders raised. Including estimated Import Duty and VAT the total is now at £740 for this upgrade, so shopping around made considerable savings. It will be interesting to see what the actual import duty and fees will work out to eventually.

**Comment by Sean-anotherone** [comment]
Great stuff.  Can make a start on some preliminary work this Thursday.   
Can I suggest adding 6ft of nice bright leds to that list, as the interior lighting is a bit ropey.

Sean

Sent from Yahoo Mail on Android 
 
  On Tue, 25 Aug 2020 at 12:32, Mike Gorman<notifications@github.com> wrote:   


With the decision being taken in the recent Directors meeting to proceed with replacing the tube on Gerald, I have built a shopping list for the parts whic is attached below.

Monty Rebuild Cost 25AUG2020.xlsx

I will work with @DoESsean on Thursday to get these orders raised. Including estimated Import Duty and VAT the total is now at £740 for this upgrade, so shopping around made considerable savings. It will be interesting to see what the actual import duty and fees will work out to eventually.

**Comment by magman2112** [comment]
With the laser tube being ordered by @JackiePease today and all of the other pieces on their way, we should have all of the new parts with us in the next couple of weeks.

**Comment by Sean-anotherone** [comment]
We should use this Thurs to agree some designs fro the few odd bits that meed to be made/modified, and a prioritised list for order of installation and testing.
I also think a local RCD will be necessary on the incoming supply, as we have water over power.
even one of these (this is just audio, something that trips to isolation would be better)
Firechief Flood Alarm Base with Probe (FC4056PB) | Zoro UK


| 
| 
| 
| £10.59 |  |

 |

 |
| 
|  | 
Firechief Flood Alarm Base with Probe (FC4056PB) | Zoro UK

Firechief - Flood Alarm Base with Probe (FC4056PB) available at www.zoro.co.uk - UK's Leading Supplier of DIY To...
 |

 |

 |



Or  :
2 metre Water Detecting Rope Control Panel|KP|


| 
| 
| 
| £121.99 |  |

 |

 |
| 
|  | 
2 metre Water Detecting Rope Control Panel|KP|

2 metre KP Water Detecting Rope with Control Panel for use in monitoring large areas for Water Leaks & early sig...
 |

 |

 |



Or one of these, hacked to trip a relay that is fitted above the wet parts.  ( actually, inverted so as to not engage a relay when wet, so it fails safe)
Water Leakage Alarm Detector 120dB Water Leak Sensors Flood Alert Overflow Al Fs | eBay


| 
| 
| 
| £4.01 |  |

 |

 |
| 
|  | 
Water Leakage Alarm Detector 120dB Water Leak Sensors Flood Alert Overfl...

Automatic operation. No buttons, no programming, just place it where you want to check water leakage;. Made of d...
 |

 |

 |







    On Sunday, 13 September 2020, 21:52:15 BST, Mike Gorman <notifications@github.com> wrote:  
 
 


With the laser tube being ordered by @JackiePease today and all of the other pieces on their way, we should have all of the new parts with us in the next couple of weeks.

**Comment by Sean-anotherone** [comment]
Spent a bit of time working on Tony this week:

Power board out, refurbished and HV psu installed

Power board refitted and all the existing stuff reconnected, with ferrules throughout (@magman2112  would be proud)


Next jobs:

Clear some space around tyhe machine - access will be needed behind and to both ends.

Set up the chiller and water flow switch (on the return pipe so it trips in event of a leak).  Do the associated plumbing and run pipes to the tube area.  Connects to "machine" MCB on the power board

Connect the lid interlock to the new PSU.

Connect the cohesion board to the PSU control input and investigate any settings in the firmware that need to be changed.

Fit the water-deflector sheet (a sheet of visor pp over the power board in case there is a water leak)

Install tube, fill coolant

align mirrors and beam combiner (need to run 5V to the red dot laser too)

fit covers back on

Set up lightburn on a suitable PC, connected by USB, and also connect one of the new laser room network cables to the cohesion board to allow network control.

Check all interlocks and the estop, etc.

train the first few users.

👍

**Comment by Sean-anotherone** [comment]
I understand there are some folks ready to assist with bringing Tony online, so I will try to give enough info below to enable some of the outstanding tasks to progress.

First task is the plumbing and hookup for the cooler:

1. The cooler unit needs to be positioned - this might fit on the frame behind the machine, that would be preferred, or it might need to be situated on the floor behind.
2. Routing for the water pipes needs to be determined - how they get inside the machine and their route internally.  Flow switch needs fitting, too, probably on the cooler.
3. The wiring for the cooler interlock (connector on real of cooler) needs working out and some cable fitting to the plug (it has been supplied).  This needs to connect to one of the safety inputs on the HV supply, and depending on whether it is a contact or an active high/low output, it might be good to wire in series with the flow switch that has been bought to provide added protection.  The flow switch needs fitting on the return line to the cooler, so it trips if a pipe comes off.

4. Holes drilled/cut for the pipes, grommets or glands installed to prevent chafing/cutting and the pipes run.  Leave the pipes long for now.

Another Task is the work at the RH end of the machine
1. fit some foam around the compressor to stop it floating about.
2. The enclosure for the interface panel is made, but it needs a plate fitting over the old opening in the RH top cover and the control fitting to that plate.
3. Need a usb lead fitting and routing so it can plug into the laptop - the plan for the laptioop is that it will sit on the wooden shelf that rests on the extract duct.
4. Further servo-tuning is required.  This might be best left for last.
5. Install and route the cable for the video camera - this goes inside the bed area, probably on the quikframe in the middle of the lid, and plugs into the laptop
6. test fit the top cover, but leave it loose (but secured) for now, to allow for (4)

A short task 

1. Prepare and solder on a cable for the lid interlock microswitches (there are 2 in series at the back left corner) and route this to the leftmost block of terminals on the power board (where the ribbon cable from the HV psu terminates). This needs to be heatshrunk etc. as it needs to be robust / rugged.  Main safety circuit.
2. Check and set the lid interlock - the sensitivity to opening needs to be set, and the spring tension adjusted to it switches on & off cleanly, plus the mechanical shutter needs checking / oiling / adjusting as necessary.

Water protection for the power board

1. take a sheet (might need 2?) of polypropylene left from the visor work and fit it such that any leakage from the laser is deflected over the electricals and onto the floor.  Foam tape, backed up with a strip of metal with nuts & bolts should suffice.


Install and align the new laser tube

There is a small beam combiner that was rescued from the RF laser, it would be good to fit this in the main beam path, but it will need manual alignment / shimming and mounts making only after the tube is in and alignment is good, or alternatively, take the beam combiner out of it's current box and make a simple adjustable mount for it.  The power requirement needs checking too - I think it's 5V but this needs confirming and the polarity determining etc. 


I am of course happy to discuss any/all the above, as I'm sure  Mike will be.

Sean

**Comment by Sean-anotherone** [comment]
I installed most of the plumbing yesterday - the chiller supply and the flow meter.  Currently there is just a small loop inside the machine for leak testing, which it did not.

I'll open a separate thread for the purchase of some RO water, which would be good to use as this tube has a long life.

**Comment by magman2112** [comment]
@Sean-anotherone mentioned that to have a chance of having a Class 1 laser classification for Tony, then a key switch must be used to enable operation of the machine.

I have ordered a reasonable quality one from Amazon, which should be delivered today, we can then fit this on Thursday hopefully when we are next in DoES. The switch fits in a 22mm hole, so will likely easily replace the broken toggle switch in the front panel.

**Comment by Sean-anotherone** [comment]
@magman2112 ad I had another good session on this machine yesterday, and is is now closer than ever.  Once the alignment is complete, the handoff for general use is about an hour away (just some interlock tests and fitting covers)

There are "wishlist" items, like homing and limit switches, and the red-dot pointer, but all the main hardware is now fully completed..

The one outstanding job is the first-time beam alignment.

There is an inconvenience in the firmware that results in the laser test-fire pulse being of 10 seconds duration (i.e. sets fire to the target), and I've posted in the cohesion forums to see if there is any way to change this, a reply awaited.

The back-up solution is a call-to-arms for one of out IOT/arduino/ESP experts :

There is a spare 6-way connector for the laser psu that can be easily configured as an interlock by-pass and alignment tool, but it needs an arduino (or anything that can run on 5V) setting up to produce a 20kHz @5V  PWM train of variable duration or variable duty (around 15mS at 50% is my opening guess), with some way to set the duration or duty as the tube ages (a pot would be fine). 

One press on a button needs to give one buzz of PWM at the set duration/duty, then wait for another press.

I'd rig this in a small box with the fire button on top and a cable long enough to plug in to the supply and be conveniently located for pressing during alignment.  This is the second choice to a firmware change, but I thought it would be good to get a willing volunteer now in case the forums yield no luck.

Sean

**Comment by magman2112** [comment]
@Sean-anotherone and i had a successful night last night and managed to get Tony cutting.

The alignment was a long winded affair, as this is the first major alignment including all of our modifications, along with the existing mirrors and lenses. Given that there are 4 levels of adjustment on the tube alone which adds to the 4 adjustments possible on each mirror, it took us many hours to get the alignment anything close to useful. We finished around 11:30pm, but with some useful results.

With the initial alignment completed, the first cuts showed the power and precision of the new machine. We successfully cut 3mm plywood at a speed of 40mm/sec at 70% power, this is 2-3 times the speed of Gerald. We are likely to set maximum power to 80% for now, to promote longevity for the tube, this is currently set in firmware and we will see how this works out.

There are still a few issues to be resolved with Tony before it is available for full use though. 

The calibration of the axes is off to a degree, a 50mm x 50 mm square measured at 51mm x 56mm. This is adjustable in the firmware for the controller though, so this will be an easy fix, we just ran out of time this time around.

We need to work out the correct focus for different materials, as well as making a gauge for users to easily set this. Focus is set manually by moving the lens, so a gauge will almost certainly be required. I expect we will set this up for 3mm, 4mm and 6mm at first.

The air assist is not switching on under software control at the moment, so the wiring needs to be checked to fix this.

We also need to adjust some firmware constraints we used for the initial setup, to speed up some of the axis movements, as well as tuning the servo motor tuning, as at least one off the servos is “singing” at the moment.

Working out how to use the software and writing some wiki articles to help new users is also a high priority. I suspect we will need to work with multiple people to come up with some good practices for this.

We may also need to get at least home limit switches installed to make the software easier to work with, but this isn’t our highest priority at the moment.

Altogether, very good progress and we now basically have a 3rd laser cutter working, with just a few caveats to permit more general use.

**Comment by KarlDunkerley** [comment]
You guys have done sterling work. Can't wait to get my mitts on it. Thank you for this.
    On Friday, 11 December 2020, 15:55:59 GMT, Mike Gorman <notifications@github.com> wrote:  
 
 


@Sean-anotherone and i had a successful night last night and managed to get Tony cutting.

The alignment was a long winded affair, as this is the first major alignment including all of our modifications, along with the existing mirrors and lenses. Given that there are 4 levels of adjustment on the tube alone which adds to the 4 adjustments possible on each mirror, it took us many hours to get the alignment anything close to useful. We finished around 11:30pm, but with some useful results.

With the initial alignment completed, the first cuts showed the power and precision of the new machine. We successfully cut 3mm plywood at a speed of 40mm/sec at 70% power, this is 2-3 times the speed of Gerald. We are likely to set maximum power to 80% for now, to promote longevity for the tube, this is currently set in firmware and we will see how this works out.

There are still a few issues to be resolved with Tony before it is available for full use though.

The calibration of the axes is off to a degree, a 50mm x 50 mm square measured at 51mm x 56mm. This is adjustable in the firmware for the controller though, so this will be an easy fix, we just ran out of time this time around.

We need to work out the correct focus for different materials, as well as making a gauge for users to easily set this. Focus is set manually by moving the lens, so a gauge will almost certainly be required. I expect we will set this up for 3mm, 4mm and 6mm at first.

The air assist is not switching on under software control at the moment, so the wiring needs to be checked to fix this.

We also need to adjust some firmware constraints we used for the initial setup, to speed up some of the axis movements, as well as tuning the servo motor tuning, as at least one off the servos is “singing” at the moment.

Working out how to use the software and writing some wiki articles to help new users is also a high priority. I suspect we will need to work with multiple people to come up with some good practices for this.

We may also need to get at least home limit switches installed to make the software easier to work with, but this isn’t our highest priority at the moment.

Altogether, very good progress and we now basically have a 3rd laser cutter working, with just a few caveats to permit more general use.

**Comment by magman2112** [comment]
Good progress again on Thursday night.

The air-assist is now working correctly, though @Sean-anotherone did waste an hour trying to diagnose the wiring till 8 remembered that I had disable some configuration items which I had though unnecessary. A quick update of the configuration and the air-assist is now working as it should.

The air assist is noticeably noisier than that on Gerald and Sophia, but we have also noticed that it gives a much cleaner cut on plywood than the other lasers and the air-assist is only on when the laser is cutting. This is one of the many peculiarities we will have to work with on Tony, as we get used to working with the new setup.

We also calibrated the X and Y axis for dimensions, which is accurate to within 0.1 of a millimetre now.

Next, we did some initial tests on focus. It would appear that 14mm is the optimum for 3mm thick material, this should also work for thinner stock as well. We did some test cuts on 6mm MDF, this cut well at 15mm/70% with a 14mm focus, but we need to do a few more tests, as I suspect the optimum focus distance might be 12 or 13mm for thicker material. Basically, because the focusing lense has a short focal length, the depth of field may play a bit more in finding the optimum focus with the best focus likely needing to be 1 or 2mm into the material. Sean cut a piece of 3mm acrylic to 14mm to use as a template for setting the focus in normal operation.

The initial limits on the machine have now also been removed, so machine movements are now a lot quicker. I expect we will have to do a little more research on the motors and motor drivers to find the optimum settings for these speeds, but what we have now are good to start user operations with.

The main outstanding issue is now installing limit switches, so that we can set a home position for the laser head, plus some tuning of the servo drives to remove the “singing”.

I plan to start working with folks to get them used to working with Tony, so that we can get people’s initial thoughts as well as finding if the machine is fit for general use at the moment. In particular, I want people other than Sean and I to start using the machine so that we can capture their experiences for the initial wiki documentation for Tony.

**Comment by Sean-anotherone** [comment]
I think it's worth adding that the machine was cutting 3mm ply at 30mm/s and power 70 - i.e. about double the speed seen on Gerald!
The lightburn software is installed and licensed, it is very intuitive but if any of the laser power-users want to draft the operating wiki that would be a big help....
The remaining jobs are :
improve the servo tuning (this will need some time and effort and a bit of research first, it works well as it is but could be better)fit homing sensors / limit sensors (inductive or hall effect - need to be specified, purchased and bracketry made to fit them.  There may be some recommendations on the cohesion forums)replace the RH top cover
I think that's all. !
Sean




    On Friday, 18 December 2020, 16:22:07 GMT, Mike Gorman <notifications@github.com> wrote:  
 
 


Good progress again on Thursday night.

The air-assist is now working correctly, though @Sean-anotherone did waste an hour trying to diagnose the wiring till 8 remembered that I had disable some configuration items which I had though unnecessary. A quick update of the configuration and the air-assist is now working as it should.

The air assist is noticeably noisier than that on Gerald and Sophia, but we have also noticed that it gives a much cleaner cut on plywood than the other lasers and the air-assist is only on when the laser is cutting. This is one of the many peculiarities we will have to work with on Tony, as we get used to working with the new setup.

We also calibrated the X and Y axis for dimensions, which is accurate to within 0.1 of a millimetre now.

Next, we did some initial tests on focus. It would appear that 14mm is the optimum for 3mm thick material, this should also work for thinner stock as well. We did some test cuts on 6mm MDF, this cut well at 15mm/70% with a 14mm focus, but we need to do a few more tests, as I suspect the optimum focus distance might be 12 or 13mm for thicker material. Basically, because the focusing lense has a short focal length, the depth of field may play a bit more in finding the optimum focus with the best focus likely needing to be 1 or 2mm into the material. Sean cut a piece of 3mm acrylic to 14mm to use as a template for setting the focus in normal operation.

The initial limits on the machine have now also been removed, so machine movements are now a lot quicker. I expect we will have to do a little more research on the motors and motor drivers to find the optimum settings for these speeds, but what we have now are good to start user operations with.

**Comment by magman2112** [comment]
Working with the machine on Saturday with @JackiePease and AJ, we worked out some cutting speeds for paper, but also noticed that lines were not cutting completely to the corners. AJ found some forum comments from the Lightburn forum which pointed to some issues with the minimum power settings in the controller configuration files. After some testing, we set this minimum to 0.06 (6%) and cutting was significantly improved.

We also noticed some unevenness in curved lines that will need further investigation. This is likely to be either a servo tuning issue or possibly the carriage bearings may need adjustment, as we were using high speeds to cut paper, which highlighted the issue. Luckily, this is repeatable, which will help with the diagnosis.

Next, we found that when jogging the carriage position with the lid down, that the laser is being switched on during the movement. This was also happening when testing the image size using the Frame button in Lightburn with the lid down. Manual jog and shuttle from the controller front panel did not switch the laser on though, which would seem to point to it being a Lightburn issue, but we will need to investigate further regardless.

I have been looking at Hall effect sensors for the limit/home switches. I found some relatively inexpensive ones on eBay from UK suppliers, I will pick a couple up a couple of these so we can work on a home switch in the near future.

**Comment by magman2112** [comment]
A point of interest, with most of the bills now in, the cost of the new laser tube and ancillaries for Tony now stands at £585.64. We were not charged import duty on a couple of the items, particularly the laser tube, which has worked out very well for us.

with all the major items covered, we only have limit switches and possibly a few bearings as possible expenses to complete the rebuild.

**Comment by amcewen** [comment]
I've got one of [these hall effect switches](https://uk.farnell.com/hamlin/55100-3m-02-a/sensor-hall-effect-mini-flange/dp/5052580) if it's any use/interest?

**Comment by Sean-anotherone** [comment]
Looks good, two would be better but the axes don't need to be identical
    On Sunday, 20 December 2020, 20:38:23 GMT, Adrian McEwen <notifications@github.com> wrote:  
 
 


I've got one of these hall effect switches if it's any use/interest?

**Comment by magman2112** [comment]
This was one of the sensors I almost bought today from Rapid Electronics. The Farnell ones are actually cheaper but they don’t have stock till July next year, so I think I will still get these from Rapid. These should also be light enough to mount on the carriage, which I now think will be necessary for some of the limit switches

**Comment by Sean-anotherone** [comment]
Easier to mount the magnet on the carriage and fit these to the chassis - otherwise got to thread a cable through 2 snakes.
I think xref should be the left side and yref the far end, to keep both of them out of the way.  I recall that the software allows the ref to be somewhere that isn't 0,0
    On Sunday, 20 December 2020, 21:20:45 GMT, Mike Gorman <notifications@github.com> wrote:  
 
 


This was one of the sensors I almost bought today from Rapid Electronics. The Farnell ones are actually cheaper but they don’t have stock till July next year, so I think I will still get these from Rapid. These should also be light enough to mount on the carriage, which I now think will be necessary for some of the limit switches

**Comment by amcewen** [comment]
I thought I had more than one, but digging out the Farnell invoice to check that only lists one.  I'll check when I'm next in and leave it somewhere for you

**Comment by magman2112** [comment]
Thanks Adrian.

Sean, if we were just going to use the switches for the home position only, we could likely get away with mounting them on the chassis. In the long run though, I think we should aim for limit switches on all the axes, which means that limit switches on the front to back movement will need to be mounted on the carriage. I suspect that these Hall effect switches will be light enough to not cause any issues if mounted on the carriage, it’s just routing the cables which will be a bit more work as you mentioned. The left and right limit switches can still be mounted on the chassis though. 

I’ve also looked at picking up some countersunk magnets to go with the sensors, to make them easier to mount.

**Comment by magman2112** [comment]
I was doing some web searches for the lasing during moving issue, I found a comment on the Lightburn forums that is something we need to test.

https://forum.lightburnsoftware.com/t/laser-not-turning-off-between-cuts-burning-during-travel-moves/8528

To paraphrase, the GRBL controller needs to be switched into Laser mode, by typing $32=1 at the console. If this fixes the issue, it is a nice easy fix.

**Comment by Sean-anotherone** [comment]
I looked at this today - our cohesion controller is currently on smoothieware, but is switching to GRBL is necessary......
Sean

    On Wednesday, 30 December 2020, 10:59:42 GMT, Mike Gorman <notifications@github.com> wrote:  
 
 


I was doing some web searches for the lasing during moving issue, I found a comment on the Lightburn forums that is something we need to test.

https://forum.lightburnsoftware.com/t/laser-not-turning-off-between-cuts-burning-during-travel-moves/8528

To paraphrase, the GRBL controller needs to be switched into Laser mode, by typing $32=1 at the console. If this fixes the issue, it is a nice easy fix.

**Comment by Sean-anotherone** [comment]
Found this :
https://embeddedtronicsblog.wordpress.com/2018/12/27/grbl-lpc-c3d-laserboard-firmware-builds/

Sean


    On Wednesday, 30 December 2020, 10:59:42 GMT, Mike Gorman <notifications@github.com> wrote:  
 
 


I was doing some web searches for the lasing during moving issue, I found a comment on the Lightburn forums that is something we need to test.

https://forum.lightburnsoftware.com/t/laser-not-turning-off-between-cuts-burning-during-travel-moves/8528

To paraphrase, the GRBL controller needs to be switched into Laser mode, by typing $32=1 at the console. If this fixes the issue, it is a nice easy fix.

**Comment by magman2112** [comment]
Time for an update.

over the last couple of weeks, @Sean-anotherone and I have made quite a few adjustments to Tony to improve its performance. 

The firmware has been upgraded to a more powerful version of Smoothieware, now with clustering. This requires a small configuration change in Lightburn to work with this version of firmware. The major change is in the performance in engraving, which is purportedly 4 time faster in this version of the firmware. Initial tests are showing good performance, but we need people to test this further and give us some feedback.

The issue with the laser turning on during movements was traced to the minimum power setting that we had set in the controller configuration at 6%, to give better performance on corner cutting. As the lasing during movement is more critical, we have reverted this setting back to 0%. Again, further testing is needed to see if this setting needs to be tweaked slightly.

Sean found a configuration setting in Lightburn that significantly improved the cutting of curves. It was actually set to it’s minimum level, now it is at its maximum. Tweaking is an option here to see what difference this makes to the speed of cuts - so more testing needed.

Last Thursday, Sean adjusted the servo motor tuning. With the current settings, there is much less “singing” of the servo’s and overshoot on engraving has effectively been eliminated. We also found some play in the carriage bearings, which was adjusted and improved.

With all of these changes, Tony is now in a good state for people to start using this laser cutter in anger. Further tuning is no doubt likely to be needed, but we need more use of the machine to build up our knowledge - and this is for all users in DoES.

Outstanding items are:-

- The installation of limit switches - which we have an initial design and plan for. 
- Obtain some new bearings for some parts of the machine, especially the x-axis carriage
- A few couplings need tightening (especially the x-axis drive wheels at the back of the machine)
- Identify, obtain and fit better lighting for illuminating the bed.
- Fit the camera to enable the Lightburn camera feature.

The main outstanding item though is to write some user instructions on how to use Tony and the new Lightburn software. Ideally this should be done by people other than Sean and I, as we suspect we have too much familiarity with the machine. I will raise this as a separate issue to highlight it.

**Comment by KarlDunkerley** [comment]
Ooooh can't wait!


    On Saturday, 27 February 2021, 14:19:33 GMT, Mike Gorman <notifications@github.com> wrote:  
 
 


Time for an update.

over the last couple of weeks, @Sean-anotherone and I have made quite a few adjustments to Tony to improve its performance.

The firmware has been upgraded to a more powerful version of Smoothieware, now with clustering. This requires a small configuration change in Lightburn to work with this version of firmware. The major change is in the performance in engraving, which is purportedly 4 time faster in this version of the firmware. Initial tests are showing good performance, but we need people to test this further and give us some feedback.

The issue with the laser turning on during movements was traced to the minimum power setting that we had set in the controller configuration at 6%, to give better performance on corner cutting. As the lasing during movement is more critical, we have reverted this setting back to 0%. Again, further testing is needed to see if this setting needs to be tweaked slightly.

Sean found a configuration setting in Lightburn that significantly improved the cutting of curves. It was actually set to it’s minimum level, now it is at its maximum. Tweaking is an option here to see what difference this makes to the speed of cuts - so more testing needed.

Last Thursday, Sean adjusted the servo motor tuning. With the current settings, there is much less “singing” of the servo’s and overshoot on engraving has effectively been eliminated. We also found some play in the carriage bearings, which was adjusted and improved.

With all of these changes, Tony is now in a good state for people to start using this laser cutter in anger. Further tuning is no doubt likely to be needed, but we need more use of the machine to build up our knowledge - and this is for all users in DoES.

Outstanding items are:-
   
   - The installation of limit switches - which we have an initial design and plan for.
   - Obtain some new bearings for some parts of the machine, especially the x-axis carriage
   - A few couplings need tightening (especially the x-axis drive wheels at the back of the machine)
   - Identify, obtain and fit better lighting for illuminating the bed.
   - Fit the camera to enable the Lightburn camera feature.

The main outstanding item though is to write some user instructions on how to use Tony and the new Lightburn software. Ideally this should be done by people other than Sean and I, as we suspect we have too much familiarity with the machine. I will raise this as a separate issue to highlight it.

**Comment by magman2112** [comment]
Last Thursday, @Sean-anotherone and I fitted some new LED lighting, that Sean found. This has significantly improved the lighting for Tony, so that is one more issue ticked off the list.

we have also got a basic plan for the installation of the limit switches, which will require some brackets to be made, but it is progress anyway.

**Comment by magman2112** [comment]
A quick update on some of the outstanding tasks for Tony.

@Sean-anotherone has made a couple of the brackets required to install the limit switches and I have built the wiring harness to be fitted to the moving rail. We will hopefully get the remaining brackets made in the next few weeks, then enable the limit switches and homing on the controller. We have already obtained the Hall effect sensors and magnets needed for these limit switches.

For the Lightburn camera, I found 3D print files for a holder for the camera we originally bought (the files are on the Lightburn site itself). I have now printed these out on my resin printer and we will look to see what else is needed to enable this facility on Tony.

**Comment by magman2112** [comment]
@Sean-anotherone and I completed the installation of the limit switches on Thursday 17th June but the still homing didn’t work correctly, as we thought the magnets weren’t strong enough. 

I picked up some more magnets from Amazon yesterday and we found that there were still problems with some of the Hall effect sensors. We then found that the Hall effect sensors were sensitive to the pole direction of the magnet. We found that the original magnets were 2 different poles, 3 of the sensors didn’t work because of this. 

One of the sets of new magnets had the correct pole direction, so all of the magnets now trigger the sensors correctly. The hall sensors now trigger approx 25mm from the magnet, which gives us some leaway for high speed approaches to the limits.

We found that people have been hitting the limits in the last week, as 2 of the magnet arms for the sensors were bent, one was actually bent backwards showing the force it had been hit with. Tony now generates a fault condition if the limits are hit, this can be cleared by completing a home command - either from Lightburn or from the front control panel.

With the homing sensors now working and some loose guides and wheels adjusted, Tony is almost 100% completed. Sean and I plan another full alignment pass next week, plus we are going to look at fitting the digital camera and its mount. After this, we just need to fit the right hand cover and build for Tony will be complete.

**Comment by magman2112** [comment]
@Sean-anotherone and I completed a full alignment of the laser tube and mirrors on Tony, which significantly improved the cutting of the laser across the whole of the bed. 

All of the covers were then installed, which is effectively the final task in commissioning this laser cutter. There are still a few tuning tasks to be undertaken and the Lightburn camera to be attached to the PC, but these can be broken out as separate issues.

so finally closing this issue.

**Comment by Sean-anotherone** [comment]
WooHoo!:)

    On Friday, 30 July 2021, 09:37:04 BST, Mike Gorman ***@***.***> wrote:  
 
 


@Sean-anotherone and I completed a full alignment of the laser tube and mirrors on Tony, which significantly improved the cutting of the laser across the whole of the bed.

All of the covers were then installed, which is effectively the final task in commissioning this laser cutter. There are still a few tuning tasks to be undertaken and the Lightburn camera to be attached to the PC, but these can be broken out as separate issues.

so finally closing this issue.

**Comment by arthurrowland** [comment]
Well done laser team! Milestone!
